Fatras/include/ActsFatras/Kernel/Simulation.hpp

```diff
@@ -49,13 +49,22 @@ struct SingleParticleSimulation {
   interactions_t interactions;
   /// Selector for surfaces that should generate hits.
   hit_surface_selector_t selectHitSurface;
+  /// Wrapped logger for debug output.
+  Acts::LoggerWrapper loggerWrapper = Acts::getDummyLogger();
   /// Local logger for debug output.
   std::shared_ptr<const Acts::Logger> localLogger = nullptr;
 
   /// Construct the simulator with the underlying propagator.
   SingleParticleSimulation(propagator_t &&propagator_, Acts::Logging::Level lvl)
       : propagator(propagator_),
-        localLogger(Acts::getDefaultLogger("Simulation", lvl)) {}
```

Comment:
Could we not modify this to accept the logger as an argument, and default it to `getDefaultLogger`? That's what we do in most cases.

Reply:
That's actually a good call, I think we should it consistent and simple.